birdhouse/optional-components/testthredds/docker-compose-extra.yml

```diff
@@ -45,3 +45,6 @@ services:
 volumes:
   testthredds_persistence: {}
   testwps_outputs: {}
+  testdatasets:
+    driver_opts:
+      device: /data/testdatasets
```

Comment:
Why not using `${DATA_PERSIST_ROOT}` rather than `/data`?
And what's the point of moving this volume as a named volume? I don't see it used anywhere else.

Reply:
I experimented with named volumes and found out that docker-compose doesn't handle them the same way and mappings in services. 
For example, it will not create the missing directory, only attempt to mount an existing one and fail if missing. 

In this case, I will move it within the `volumes` of `testthredds` with the variable since a path location is specified. 
Named volumes are useful now only when `{}` is used to let docker manage it automatically (but volatile if removed).